### PR TITLE
Add support for webhooks

### PR DESCRIPTION
**Pull request by eranelbaz**
Add support to generate `openapi` with `webhooks` property
It will search for it 
```
custom:
  documentation:
    webhooks:
      webhookName:
        post:
          requestBody:
            description: |
              This is a request body description
          responses:
            200:
              description: |
                This is a expected response description
```

### CODE REVIEW

**Comment by roni-frantchi** [comment]
Thanks for contributing @eranelbaz - can you please add a descriptive entry in the readme as to how this works?  
An example of not just the input - but also what would be produced and when it should be used.  
See other features for example

File: src/serverless-openapi-typescript.ts [hunk 1]
```diff
@@ -1,274 +1,302 @@
-import type Serverless from "serverless";
-import fs from "fs";
-import yaml from "js-yaml";
-import {SchemaGenerator, createGenerator} from "ts-json-schema-generator";
-import {upperFirst, camelCase, mergeWith, set, isArray, get, isEmpty, kebabCase} from "lodash" ;
-import {ApiGatewayEvent} from "serverless/plugins/aws/package/compile/events/apiGateway/lib/validate";
-import { mapKeysDeep, mapValuesDeep} from 'deepdash/standalone'
```
Comment (roni-frantchi) [inline src/serverless-openapi-typescript.ts:7]: @eranelbaz please clear this PR from all style changes or changes not related to the proposed feature

**Review by roni-frantchi — APPROVED** [review]
Nice!

**Comment by roni-frantchi** [comment]
@eranelbaz please attend to [this](https://github.com/env0/serverless-openapi-typescript/pull/47#issuecomment-2112356945) before merging - also see test failures

**Comment by eranelbaz** [comment]
Fixed test + added reamde

**Pull request merged**